webapp/portlet/src/main/webapp/external-spaces-list/components/ExternalSpacesRequestsItems.vue

```diff
@@ -0,0 +1,125 @@
+<template>
```

Comment:
I think that it is better to to add elementary SpaceRequestItem.vue module instead